SpaceCore/SpaceCore.cs

```diff
@@ -94,6 +97,21 @@ private void OnUpdate(object sender, UpdateTickedEventArgs e)
             }
         }
 
+        private void OnOneSecondUpdateTicked(object sender, OneSecondUpdateTickedEventArgs e) {
+            if (this.NextToDispose != null)
+                {
+                this.NextToDispose.Dispose();
+                this.NextToDispose = null;
+                }
+            if (this.Config.DisposeOldTextures)
+                {
+                // TryDequeue not available on net452
+                if (TextureDisposalQueue.Count != 0)
+                    // Dispose exactly one item. We don't want too-early disposal.
+                    this.NextToDispose = TextureDisposalQueue.Dequeue();
+                }
+            }
+
```

Comment:
We don't dispose directly from `TextureDisposalQueue` to give at least 1 second delay between Texture becoming unused and actual disposal.